crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -1296,8 +1298,10 @@ module deepbook::clob {
                 price_low,
                 clock::timestamp_ms(clock)
             );
-            vector::push_back(&mut price_vec, price_low);
-            vector::push_back(&mut depth_vec, depth);
+            if (depth != 0) {
```

Comment:
Can we add a specific unit test here that shows what happens if the `depth == 0` ?